contracts/StreamManager.sol

```diff
@@ -3,17 +3,26 @@ pragma solidity ^0.8.0;
 
 import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
 import "../interfaces/IStreamManager.sol";
+import "../interfaces/IOpenStream.sol";
 import "./OpenStream.sol";
 
 contract StreamManager is IStreamManager {
     using SafeERC20 for IERC20;
 
+    /// @dev Mapping for addresses of streams instance 
+    mapping(address => address) public streams;
     /**
      * @dev New open stream event
      * @param _payer payer address
      * @param _itself open steam instance address
      */
     event OpenStreamCreated(address _payer, address _itself);
+    /**
+     * @dev Cancel open stream event
+     * @param _payer payer address
+     * @param _amount amount of the tokens
+     */
+    event CancelStreamCreated(address _payer, uint256 _amount);
```

Comment:
what does `CancelStreamCreated` mean? It is supposed to be `CancelStream`?
@CTAPCKPIM